formats/publication/frontend/examples/withdrawn_publication.json

```diff
@@ -0,0 +1,58 @@
+    ],
+    "first_public_at": "2011-09-29T01:00:00+01:00",
+    "change_history": [
+      {
+        "public_timestamp": "2013-09-06T10:11:28+01:00",
+        "note": "Published updated pig keepers guide."
+      },
+      {
+        "public_timestamp": "2011-09-29T01:00:00+01:00",
+        "note": "First published."
+      }
+    ],
+    "tags": {
+      "browse_pages": [],
+      "policies": [],
+      "topics": []
+    },
+    "government": {
+      "title": "2010 to 2015 Conservative and Liberal Democrat coalition government",
+      "slug": "2010-to-2015-conservative-and-liberal-democrat-coalition-government",
+      "current": false
+    },
+    "political": false,
+    "withdrawn_notice": {
+      "explanation": "<div class=\"govspeak\"><p>This information is now out of date. For our current guidance please read <a href=\"https://www.gov.uk/government/collections/guidance-for-keepers-of-sheep-goats-and-pigs\">guidance for keepers of sheep, goats and pigs</a>.</p></div>",
+      "withdrawn_at": "2015-01-13T13:05:30Z"
+    }
+  },
+  "links": {
+    "lead_organisations": [
+      {
+        "content_id": "de4e9dc6-cca4-43af-a594-682023b84d6c",
+        "title": "Department for Environment, Food & Rural Affairs",
+        "base_path": "/government/organisations/department-for-environment-food-rural-affairs",
+        "api_url": "https://www.gov.uk/api/content/government/organisations/department-for-environment-food-rural-affairs",
+        "web_url": "https://www.gov.uk/government/organisations/department-for-environment-food-rural-affairs",
+        "locale": "en",
+        "analytics_identifier": "D7"
+      }
+    ]
+  },
+  "schema_name": "publication",
+  "document_type": "publication"
```

Comment:
I know this is deliberately left for later, but if you specify the correct key here (`guidance`) and for the Veolia publication (`notice`), we get 50% of that work done.